### ISSUE TITLE
[Accessibility]: Commits page - Message alt text is duplicated

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#34976]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34423

### Problem description

<img width="565" alt="image" src="https://user-images.githubusercontent.com/9516420/166910328-0246a40f-def8-4b7f-9e0d-eddca5f39d44.png">

The alt text for this message reads both the commit message AND the full commit body. This is duplicated, as the user can they click the "..." icon to expand the body and read it again.

### Expected behavior

Alt text should only read the commit message, as shown in the UI.

Screen reader users can still use the "..." icon to access the full body

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
